src/main/java/frc/robot/commands/DrivetrainPIDTurnDelta.java

```diff
@@ -58,6 +58,14 @@ public DrivetrainPIDTurnDelta(double deltaDegree, double forwardSpeed) {
         double currentAngle = drivebase.getCurrentAngle();
         logger.addHandler(Config.logFileHandler);
 
+        if(_pidgey == null){
+
```

Comment:
What is best practice in the robot code if components are unavailable? Log it at least in my opinion.

Reply:
Thats true, Ill look into it